build.gradle

```diff
@@ -375,7 +376,8 @@ dependencies {
   annotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
 
   implementation group: 'org.projectlombok', name: 'lombok', version: versions.lombok
-  implementation "ch.qos.logback:logback-classic:1.2.6"
+  implementation group: 'ch.qos.logback', name: 'logback-classic', version: versions.logbackVersion
+  implementation group: 'ch.qos.logback', name: 'logback-core', version: versions.logbackVersion
```

Comment:
why do we need core?

Reply:
Otherwise dependency check fails

Reply:
why it was working without it?

Reply:
be cause of the suppression of the vulnerability CVE-2021-42550

Reply:
it can not be the main reason. I guess it brings another lib an overrides other one